.gitignore

```diff
@@ -1,4 +1,6 @@
 .psci
 .psci_modules
+.history
```

Comment:
This is a folder used by VSC addon (Local history) to store files history.
I miss this functionality from Idea.